pkg/bindings/kube/types.go

```diff
@@ -1,12 +1,12 @@
-package play
+package kube
 
 import (
 	"net"
 )
 
-//go:generate go run ../generator/generator.go KubeOptions
-// KubeOptions are optional options for replaying kube YAML files
-type KubeOptions struct {
+//go:generate go run ../generator/generator.go PlayOptions
+// PlayOptions are optional options for replaying kube YAML files
+type PlayOptions struct {
```

Comment:
This is a breaking change - current users couldn't update to a newer version.

Please use type aliasing as suggested in a previous comment.